cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/сonsentStatus/ConsentStatusParamRequest.kt

```diff
@@ -0,0 +1,39 @@
+package com.sourcepoint.cmplibrary.data.network.model.optimized.сonsentStatus
+
+import com.sourcepoint.cmplibrary.data.network.util.Env
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.JsonObject
+
+@Serializable
+internal data class ConsentStatusParamRequest(
+    @SerialName("env") val env: Env,
+    @SerialName("metadata") val metadata: String,
+    @SerialName("propertyId") val propertyId: Long,
+    @SerialName("accountId") val accountId: Long,
+    @SerialName("authId") val authId: String?,
+    @SerialName("localState") val localState: JsonElement?,
+    @SerialName("includeData") val includeData: JsonObject,
+)
+
+enum class GranularState {
```

Comment:
Maybe `GranularState` and `GCMStatus` should be moved to their own files if we remove the class above. And this file can be deleted.